src/Analysis/GWASV2/Components/SelectCohort/SelectCohortDropDown.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useQuery, queryConfig } from 'react-query';
+import { Select, Spin } from 'antd';
+import { fetchCohortDefinitions } from '../../Shared/cohortMiddlewareApi';
+import { useFetch } from '../../Shared/formHooks';
+import { useSourceContext } from '../../Shared/Source';
+
+const SelectCohortDropDown = ({
+  selectedCohort = undefined,
+  handleCohortSelect,
+}) => {
+  const { source } = useSourceContext();
+  const cohorts = useQuery(
+    ['cohortdefinitions', source],
+    () => fetchCohortDefinitions(source),
+    queryConfig,
+  );
+  const fetchedCohorts = useFetch(cohorts, 'cohort_definitions_and_stats');
+
+  const onChange = (selectedCohortDefinitionId) => {
+    console.log(`selected ${selectedCohortDefinitionId}`);
+    // find cohort object based on id:
+    const selectedCohort = fetchedCohorts.find(item => item.cohort_definition_id === selectedCohortDefinitionId);
+    handleCohortSelect(selectedCohort);
+  };
+
+  const onSearch = (value) => {
+    console.log('search:', value);
+  };
+
+  return cohorts?.status === 'success' ? (
+  <Select
+    style={{width: '300px'}}
+    showSearch
+    placeholder="Select a cohort"
+    optionFilterProp="children"
+    onChange={onChange}
+    onSearch={onSearch}
+    dropdownStyle={{minWidth: '800px'}}
```

Comment:
inline style woohoo nice 😊

Reply:
nice work Pieter the dropdown looks good too

Reply:
@alilahrime to be clear: this code does _**not**_ mean I'm endorsing inline styling! I'm counting on the fact that most styling tickets will be picked up by someone else at the end, who will align the look and feel with the mockups made by the designer.